src/planscape/forsys/views.py

```diff
@@ -8,24 +8,44 @@
 from boundary.models import BoundaryDetails
 from conditions.models import BaseCondition, Condition, ConditionRaster
 from django.conf import settings
-from django.contrib.gis.gdal import CoordTransform, SpatialReference, GDALRaster
+from django.contrib.gis.gdal import CoordTransform, Envelope, GDALRaster, SpatialReference
 from django.contrib.gis.geos import GEOSGeometry, Point, Polygon
+from django.db.models.query import QuerySet
 from django.http import (HttpRequest, HttpResponse, HttpResponseBadRequest,
                          JsonResponse, QueryDict)
 
-
-# Configure global logging.
+# Configures global logging.
 logger = logging.getLogger(__name__)
 
+# Fetches input parameters for the scenario_set api.
+def fetch_input_params() -> dict:
```

Comment:
Indentation looks non-standard (2 spaces instead of 4)?  Please run "Format document" in VSCode.

Reply:
Fixed.

I was depending on vim for a little longer than necessary.